### ISSUE TITLE
Empty post and "unpostable" community (unreprodictable)

### ISSUE THREAD

**Issue by crosser** [movim/movim#791]
I cannot reproduce it and cannot demonstrate it (it happened on a private pod/server).

Under some unknown conditions (could be experimenting with pubsub outside of movim), a post appeared in the community with empty subject and body. In the stream, it is just displayed. When I try to open the post, I see an empty page with cloud spinner and the orange count of news in the left column appears with the number '1' and when I return to the stream I see the the creation time of the post changed to the time when I opened it. When I open this community, I see a toast message "post deleted" at the bottom. When I try to create a new post in the community (pencil button at the bottom), I get a toast message "you cannot publish a post here" and the editor page does not appear. (I am the owner of the community and there are posts there that I created earlier.)

Basically this community is stuck.

If there are suggestions where to look, I can do that and report the results.

**Comment by edhelas** [comment]
I was not able to reproduce this issue so far. Closing it.
